zubhub_frontend/zubhub/src/assets/js/styles/components/activity/activityStyle.js

```diff
@@ -81,7 +81,7 @@ export const style = theme => ({
   },
   projectsCount: {
     color: 'white',
```

Comment:
color white needs to be replaced too, if the background color is light the text is not readable.

May be we should show a preview on how text looks like with chosen backgrounds? There are color contrast testing tools we can use to show if it effects visibility/accessibility
cc @tuxology